examples/rosenbrock.py

```diff
@@ -41,51 +34,15 @@ def rosenbrock_4d(cfg, seed):
     
     return(val)
 
+# debug output
+logging.basicConfig(level=10)
 logger = logging.getLogger("Optimizer") # Enable to show Debug outputs
-logger.parent.level = 20 #info level:20; debug:10
-
-# build Configuration Space which defines all parameters and their ranges
-# to illustrate different parameter types, 
-# we use continuous, integer and categorical parameters
-cs = ConfigurationSpace()
-x1 = UniformFloatHyperparameter("x1", -5, 5, default=5)
-cs.add_hyperparameter(x1)
-x2 = UniformIntegerHyperparameter("x2", -5, 5, default=5)
-cs.add_hyperparameter(x2)
-x3 = CategoricalHyperparameter(
-    "x3", [5, 2, 0, 1, -1, -2, 4, -3, 3, -5, -4], default=5)
-cs.add_hyperparameter(x3)
-x4 = UniformIntegerHyperparameter("x4", -5, 5, default=5)
-cs.add_hyperparameter(x4)
-
-# SMAC scenario object
-scenario = Scenario({"run_obj":"quality", # we optimize quality (alternative runtime)
-                     "runcount-limit": 200, # at most 200 function evaluations
-                     "cs": cs, # configuration space
-                     "deterministic": "true",
-                     #"tuner-timeout" : 1,
-                     #"wallclock_limit": 2
-                     })
-stats = Stats(scenario)
-
-# register function to be optimize
-taf = ExecuteTAFunc(rosenbrock_4d, stats, run_obj='quality')
-
-# example call of the function
-# it returns: Status, Cost, Runtime, Additional Infos
-def_value = taf.run( cs.get_default_configuration() )[1]
-print("Default Value: %.2f" %(def_value))
 
-# Optimize
-smbo = SMBO(scenario=scenario, tae_runner=taf,
-            stats=stats,
-            rng=np.random.RandomState(42))
-try:
-    smbo.run(max_iters=999)
-finally:
-    smbo.stats.print_stats()
-print("Final Incumbent: %s" %(smbo.incumbent))
 
-inc_value = taf.run( smbo.incumbent )[1]
-print("Optimized Value: %.2f" %(inc_value))
+smac = FuncSMAC(func=rosenbrock_4d,
+                 x0 = [-1,0,1,0],
+                 upper_bounds= [1,1,1,1],
+                 lower_bounds= [-1,-1,-1,-1],
```

Comment:
Description of rosenbrock says: "All input parameters are continuous, but we will pretent that x2, x3, and x4 can only take integral values." Is that still true? If yes, I don't see where that is defined and the example needs more documentation.